### PR TITLE
5.6.1

### PR DESCRIPTION
**Pull request by bmacnaughton**
Updates to docs.

### CODE REVIEW

**Comment by bmacnaughton** [comment]
@cheempz - because this is documentation changes, primarily technical in nature, I'd like you to take a quick look to make sure that I'm not saying anything that we don't want to.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,28 +118,81 @@ The development process thus far has involved maintaining separate branches
...
+git checkout n.n.n     # the branch name
+npm install
+npm test               # presumes the docker environment for testing (see above).
+```
+
+If the change is significant you may need to run the support matrix again. That's beyond
+the scope of this document but involves testing appoptics against each released version
+of each package.
+
+An an extra precaution it is useful to test this in a real-world test harness. I use
```
Comment (cheempz) [inline CONTRIBUTING.md]: Typo: `As an extra..`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -118,28 +118,81 @@ The development process thus far has involved maintaining separate branches
...
+https://github.com/bmacnaughton/todomvc-mongodb. In the the `clean-apm` directory use
+`npm pack` to make a `.tgz` file, copy that to the `todomvc-mongodb` directory, change
+`package.json` for the `appoptics-apm` dependency to reference the `.tgz` file, install
+using `npm install`, setup the environment with `. env.sh stg` (works against the staging
+server - you might use another key), and run the server with `node server.js --fe_ip=localhost:port`.
+Use curl or a browser to execute requests against the server then check the backend to make
```
Comment (cheempz) [inline CONTRIBUTING.md]: Is it clearer to say `then check the AppOptics dashboard to make sure...`

Comment (bmacnaughton) [reply CONTRIBUTING.md]: exactly why I wanted a review; I didn't have the right term in mind. much better - thanks.

**Pull request merged**